packages/wonder-blocks-core/src/util/focus.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * List of elements that can be focused
+ * @see https://www.w3.org/TR/html5/editing.html#can-be-focused
+ */
+const FOCUSABLE_ELEMENTS =
+    'button, [href], input, select, textarea, [tabindex]:not([tabindex="-1"])';
```

Comment:
I realize this probably already existed...but should `[contenteditable]` go on this list? And maybe the utility could all interactive elements for `tabindex="-1"` and filter them out if present?

Reply:
Thanks for the suggestion! I made a note of this in WB-1920 so we can make the utility more robust and test it works as expected with the other components that would use this utility too! 

@marcysutton Could you elaborate on the second part? I'll clarify it in the ticket as well:

> And maybe the utility could all interactive elements for tabindex="-1" and filter them out if present?

Reply:
Sorry for the delay on this... my question was around whether elements removed from the tab order with `tabIndex="-1"` should still be considered focusable. For example:

```html
<a href="#" tabIndex="-1"></a>
<button tabIndex="-1"></button>
```
These elements are technically focusable with JavaScript, but not manually with the keyboard. 

Also, hidden inputs:
```
<input type="hidden" />
```

Curious to hear your thoughts on the tabIndex part in particular!

Reply:
Great points! I added these considerations to the ticket as well.

Hmmm I'm not sure about the tabIndex part yet, I think it will depend on how we use the utility. For the Tabs component, I think we shouldn't include elements with tabIndex=-1 since users can't focus on it manually with the keyboard. So if there's nothing inside a tab panel that the user can focus on themselves, then the tabpanel should be made focusable so it's in the tab order! Let me know what you think 😄

Reply:
Makes sense! Looking at this line of code now, I see how it would depend on the way the utility is used (as you said). Like, are those selectors exclusively checked or does the `[tabindex]:not([tabindex="-1"])` part apply to the other selectors, in the end? Lots of test permutations to write for that kind of utility... it would be a really handy thing to have in our toolbox!